learn/getting-started/getting-started-with-ballerina.md

```diff
@@ -0,0 +1,162 @@
+
+If you are interested in learning more about the language, read our [Language Basics](https://ballerina.io/learn/language-basics/) guide. 
+
+Let’s run this project by running `bal run` in your terminal:
+
+```
+$ bal run
+Compiling source
+	example/greeter:0.1.0
+
+Running executable
+
+Hello, World!
+```
+
+You can also generate an executable program with `bal build`:
+
+```$ bal buildCompiling source	sameera/greeter:0.1.0
+Generating executable	target/bin/greeter.jar
+```
+
+Then, use `bal run` as follows.
+
+```
+$ bal run target/bin/greeter.jar
+Hello, World!
+```
+
+## Write a simple REST API
+
+Now let’s change our greeter application to a REST API. Ballerina has first-class abstractions for services, resources, etc., and they make network service development easier and more fun. 
+
+You can replace the `main.bal` content with the following code.
+
+```
+import ballerina/http;
+
+listener http:Listener httpListener = new (8080);
+
+service / on httpListener {
+    resource function get greeting() returns string { 
+        return "Hello, World!"; 
+    }
+
+    resource function get greeting/[string name]() returns string { 
+        return "Hello " + name; 
+    }
+}
+```
+
+Let’s take a moment to digest new constructs in this code:
+
+- The `[http](https://lib.ballerina.io/ballerina/http/latest)` module provides high-level abstractions to work with the HTTP protocol. 
+- The listener declaration creates a new HTTP listener with port 8080. Listener is the entity that receives the network input and then routes it to the attached service(s).
+- The service declaration specifies the listener, to which the service gets attached, and a collection of remotely accessible methods. There are two kinds of methods: `resource` and `remote` methods.
+- Services use remote methods to expose services in procedural style: remote methods are named by verbs, and resource methods for data-oriented protocols: resources are named by nouns.
+- In this example, there are two resource methods: The first one responds to `HTTP` `GET` requests with the path `/greeting`, and the other one responds to `GET` requests with the path `/greeting/{name}`.
+- These resource methods return a `string` value that maps to the content-type `text/plan` in the `HTTP` response.
+
+
+
+If you are interested in learning more about services, read our [Network Interaction](https://ballerina.io/learn/distinctive-language-features/network-interaction/) guide. 
+
+Let’s run this project in your terminal:
+
+```
+$ bal run
+Compiling source
+	example/greeter:0.1.0
+
+Running executable
+```
+
+In another terminal window, try out the following commands:
+
+```
+$ curl localhost:8080/greeting
+Hello, World!
+
+$ curl localhost:8080/greeting/Ballerina
+Hello Ballerina
+```
+
+## Do more with Ballerina
+
+In this short tutorial, you set up your development environment and wrote two Ballerina applications.
+
+To learn more about Ballerina, take a look at the following tutorials:
```

Comment:
```suggestion
To learn more about Ballerina, take a look at the following tutorials:
```
This is more or less a `What's Next` section, which we decided to remove from Learn pages.

Reply:
I think, we need to have what's next in some situations. This is the getting started guide. Users may want to do more with Ballerina. This point may not be applicable for other guides in the `guides` section